packages/analytics-js/__mocks__/fixtures.ts

```diff
@@ -66,6 +66,61 @@ const dummyInitOptions = {
   enable: true,
 };
 
+const dummySourceConfigResponse = {
```

Comment:
Can you make sure we're not using any actual IDs here?

Reply:
Done

Reply:
The values still look real. Instead, mask them like `validLookingID123`, `dummyID` etc.

Reply:
Done